generators/Templates/_AssertEqualWithin.liquid

```diff
@@ -1 +1 @@
-{{ Sut }} |> should (equalWithin 0.000000001) {{ Expected }}
\ No newline at end of file
+{{ Sut }} |> should (equalWithin 0.01) {{ Expected }}
```

Comment:
What's the reason for this change? Just to not see as many 0s?

Reply:
Correct!